.bazelrc

```diff
@@ -1,7 +1,9 @@
+startup --windows_enable_symlinks
+build --enable_runfiles
 build --action_env=BAZEL_LINKLIBS=-l%:libstdc++.a:-lm
 build --action_env=BAZEL_LINKOPTS=-static-libgcc
 build --action_env=CUDA_DIR=/usr/local/cuda
-build --incompatible_strict_action_env --cxxopt=-std=c++17 --host_cxxopt=-std=c++17 --client_env=BAZEL_CXXOPTS=-std=c++17
+build --incompatible_strict_action_env --cxxopt=/std:c++17 --host_cxxopt=/std:c++17 --client_env=BAZEL_CXXOPTS=/std:c++17
```

Comment:
not sure the best practice here, @mavenlin can you take a look?

Reply:
I remember tensorflow use the same mechanism, but they have some fancy config script to detect cuda and generate the bazelrc.